Add changelog for v1.6.0 (#1020)
auth0/terraform-provider-auth0: CHANGELOG.md

## v1.6.0

FEATURES:
- `resource/auth0_prompt_screen_partial`: Add new resource to manage customized sign up and login experience. (1:1) ([#1013](https://github.com/auth0/terraform-provider-auth0/pull/1013))
- `resource/auth0_prompt_screen_partials`: Add new resource to manage customized sign up and login experience. (1:many) ([#1013](https://github.com/auth0/terraform-provider-auth0/pull/1013))

> - `data_source/auth0_prompt_screen_partials`: Add new data source to retrieve screen partials. ([#1013](https://github.com/auth0/terraform-provider-auth0/pull/1013))

Reviewer: ```suggestion
- `data_source/auth0_prompt_screen_partials`: Add new data source to retrieve prompt screen partials. ([#1013](https://github.com/auth0/terraform-provider-auth0/pull/1013))
```

- ENHANCEMENTS:
- `resource/auth0_client`: Add Organizations for Client Credentials. ([#1009](https://github.com/auth0/terraform-provider-auth0/pull/1009))

> - `resource/auth0_prompt_custom_text`: Add support for languages being fetched from CDN during Universal Login. ([#1006](https://github.com/auth0/terraform-provider-auth0/pull/1016))  

Reviewer: ```suggestion
- `resource/auth0_prompt_custom_text`: Add support for fetching the language list from a CDN for prompt custom text. ([#1006](https://github.com/auth0/terraform-provider-auth0/pull/1016))
```

BUG FIXES:
- `resource/auth0_connection`: Remove MinItems validation for precedence. ([#1017](https://github.com/auth0/terraform-provider-auth0/pull/1017))

NOTES:

> - `resource/auth0_prompt_partials`: Deprecated the resource. ([#1013](https://github.com/auth0/terraform-provider-auth0/pull/1013))

Reviewer: ```suggestion
- `resource/auth0_prompt_partials`: Deprecated in favor of `resource/auth0_prompt_screen_partial` and `resource/auth0_prompt_screen_partials`. ([#1013](https://github.com/auth0/terraform-provider-auth0/pull/1013))
```
